Dockerfile

```diff
@@ -0,0 +1,15 @@
+ARG PYTHON_VERSION=3.12
+FROM python:${PYTHON_VERSION}-slim
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt .
+RUN pip install -r requirements.txt
+
+COPY . .
+
+RUN python manage.py migrate
+
+CMD ["python", "manage.py", "runserver", "0.0.0.0:8080"]
```

Comment:
![image](https://github.com/mate-academy/devops_todolist/assets/87720583/b8cd3170-bdec-496e-8514-b16510d45832)
Add Eof(End-of-file), need to add an empty line at the end